[I 120906 14:35:38 web:1447] 304 GET /register (192.168.32.17) 3.66ms
[I 120906 14:35:45 web:1447] 302 GET /twitter_login (192.168.32.17) 322.44ms
[I 120906 14:35:48 handlers:147] twitter oauth_token is pnN5oSmWwdIQ3VGmYaPXsbUiSBrZuuFmWBJGJ4eqo
[I 120906 14:35:49 handlers:155] twitter Auth worked
[I 120906 14:35:49 handlers:69] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:35:49 web:1447] 302 GET /twitter_login?oauth_token=pnN5oSmWwdIQ3VGmYaPXsbUiSBrZuuFmWBJGJ4eqo&oauth_verifier=bqpJ5OmkPnc4hF4naW9by6Y0fQaNZ2BFyxRiAyCF5w (192.168.32.17) 643.98ms
[I 120906 14:35:49 handlers:209] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 14:35:49 web:1447] 200 GET /hello (192.168.32.17) 1.46ms
[I 120906 14:35:55 web:1447] 302 GET /logout (192.168.32.17) 0.33ms
[I 120906 14:35:55 web:1447] 302 GET /login (192.168.32.17) 0.60ms
[I 120906 14:35:55 web:1447] 304 GET / (192.168.32.17) 8.35ms
[I 120906 14:35:56 web:1447] 304 GET /view/504429ff41ab10602fb5e770.jpg?_re=200x300 (192.168.32.17) 59.89ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e776.jpg?_re=200x300 (192.168.32.17) 61.31ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e773.jpg?_re=200x300 (192.168.32.17) 66.44ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e77c.jpg?_re=200x300 (192.168.32.17) 59.48ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e779.jpg?_re=200x300 (192.168.32.17) 60.80ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e77f.jpg?_re=200x300 (192.168.32.17) 60.22ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e782.jpg?_re=200x300 (192.168.32.17) 67.28ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e788.jpg?_re=200x300 (192.168.32.17) 65.98ms
[I 120906 14:35:56 web:1447] 304 GET /view/50442a0041ab10602fb5e785.jpg?_re=200x300 (192.168.32.17) 64.39ms
[I 120906 14:35:57 web:1447] 304 GET /view/50442a0041ab10602fb5e78b.jpg?_re=200x300 (192.168.32.17) 274.32ms
[I 120906 14:35:57 web:1447] 304 GET /view/50442a0041ab10602fb5e78f.jpg?_re=200x300 (192.168.32.17) 270.80ms
[I 120906 14:35:57 web:1447] 304 GET /view/50442a0041ab10602fb5e793.jpg?_re=200x300 (192.168.32.17) 25.17ms
[I 120906 14:35:57 web:1447] 304 GET /view/50442a0041ab10602fb5e796.jpg?_re=200x300 (192.168.32.17) 27.64ms
[I 120906 14:35:57 web:1447] 304 GET /view/50442a0041ab10602fb5e799.jpg?_re=200x300 (192.168.32.17) 68.74ms
[I 120906 14:35:57 web:1447] 304 GET /view/50442a0141ab10602fb5e79d.jpg?_re=200x300 (192.168.32.17) 271.93ms
[I 120906 14:35:58 web:1447] 304 GET /view/50442a0141ab10602fb5e7a1.jpg?_re=200x300 (192.168.32.17) 273.98ms
[I 120906 14:35:58 web:1447] 304 GET /view/50442a0141ab10602fb5e7a5.jpg?_re=200x300 (192.168.32.17) 267.59ms
[I 120906 14:35:58 web:1447] 304 GET /view/50442a0141ab10602fb5e7a8.jpg?_re=200x300 (192.168.32.17) 276.46ms
[I 120906 14:35:58 web:1447] 304 GET /view/50442a0141ab10602fb5e7ac.jpg?_re=200x300 (192.168.32.17) 69.05ms
[I 120906 14:35:58 web:1447] 304 GET /view/50442a0241ab10602fb5e7af.jpg?_re=200x300 (192.168.32.17) 14.34ms
[I 120906 14:35:58 web:1447] 304 GET /static/images/bar.png (192.168.32.17) 0.97ms
[I 120906 14:35:58 web:1447] 304 GET /static/images/buttonbg.png (192.168.32.17) 0.39ms
[I 120906 14:35:58 web:1447] 304 GET /static/images/loginArrow.png (192.168.32.17) 0.37ms
[I 120906 14:36:00 web:1447] 304 GET /register (192.168.32.17) 0.98ms
[I 120906 14:36:06 web:1447] 302 GET /twitter_login (192.168.32.17) 271.86ms
[I 120906 14:36:09 handlers:147] twitter oauth_token is IBztmYxq6sR1lxR3k2p9hlKjSyxcRh5Q3gehNLjukU
[I 120906 14:36:10 handlers:155] twitter Auth worked
[I 120906 14:36:10 handlers:69] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:36:10 web:1447] 302 GET /twitter_login?oauth_token=IBztmYxq6sR1lxR3k2p9hlKjSyxcRh5Q3gehNLjukU&oauth_verifier=ExBzhHd6MWrAgh17XRGQDzbrabr7ZWOWSgwY0xU2s (192.168.32.17) 572.25ms
[I 120906 14:36:10 handlers:209] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 14:36:10 web:1447] 304 GET /hello (192.168.32.17) 1.42ms
[I 120906 14:36:27 web:1447] 302 GET /logout (192.168.32.17) 0.32ms
[I 120906 14:36:27 web:1447] 302 GET /login (192.168.32.17) 0.41ms
[I 120906 14:36:27 web:1447] 304 GET / (192.168.32.17) 8.21ms
[I 120906 14:36:27 web:1447] 304 GET /view/50442a0041ab10602fb5e773.jpg?_re=200x300 (192.168.32.17) 65.78ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e776.jpg?_re=200x300 (192.168.32.17) 59.75ms
[I 120906 14:36:28 web:1447] 304 GET /view/504429ff41ab10602fb5e770.jpg?_re=200x300 (192.168.32.17) 57.13ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e779.jpg?_re=200x300 (192.168.32.17) 60.67ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e77c.jpg?_re=200x300 (192.168.32.17) 58.70ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e77f.jpg?_re=200x300 (192.168.32.17) 59.79ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e782.jpg?_re=200x300 (192.168.32.17) 67.32ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e785.jpg?_re=200x300 (192.168.32.17) 64.68ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e788.jpg?_re=200x300 (192.168.32.17) 267.33ms
[I 120906 14:36:28 web:1447] 304 GET /view/50442a0041ab10602fb5e78b.jpg?_re=200x300 (192.168.32.17) 274.69ms
[I 120906 14:36:29 web:1447] 304 GET /view/50442a0041ab10602fb5e78f.jpg?_re=200x300 (192.168.32.17) 270.87ms
[I 120906 14:36:29 web:1447] 304 GET /view/50442a0041ab10602fb5e799.jpg?_re=200x300 (192.168.32.17) 270.42ms
[I 120906 14:36:29 web:1447] 304 GET /view/50442a0041ab10602fb5e793.jpg?_re=200x300 (192.168.32.17) 25.16ms
[I 120906 14:36:29 web:1447] 304 GET /view/50442a0041ab10602fb5e796.jpg?_re=200x300 (192.168.32.17) 27.87ms
[I 120906 14:36:29 web:1447] 304 GET /view/50442a0141ab10602fb5e79d.jpg?_re=200x300 (192.168.32.17) 270.82ms
[I 120906 14:36:30 web:1447] 304 GET /view/50442a0141ab10602fb5e7a1.jpg?_re=200x300 (192.168.32.17) 274.03ms
[I 120906 14:36:30 web:1447] 304 GET /view/50442a0141ab10602fb5e7a5.jpg?_re=200x300 (192.168.32.17) 267.88ms
[I 120906 14:36:30 web:1447] 304 GET /view/50442a0141ab10602fb5e7a8.jpg?_re=200x300 (192.168.32.17) 276.26ms
[I 120906 14:36:30 web:1447] 304 GET /view/50442a0141ab10602fb5e7ac.jpg?_re=200x300 (192.168.32.17) 69.36ms
[I 120906 14:36:30 web:1447] 304 GET /view/50442a0241ab10602fb5e7af.jpg?_re=200x300 (192.168.32.17) 14.32ms
[I 120906 14:36:30 web:1447] 304 GET /register (192.168.32.17) 1.11ms
[I 120906 14:44:26 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 14:44:31 web:2027] Invalid cookie signature 'InllcG4i|1346909770|489d254e5689b8f9364d367bbf1cf3f1dfa6c49c'
[I 120906 14:44:31 web:1447] 304 GET /register (192.168.32.17) 4.11ms
[I 120906 14:47:22 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 14:47:32 web:2027] Invalid cookie signature 'InllcG4i|1346909770|489d254e5689b8f9364d367bbf1cf3f1dfa6c49c'
[I 120906 14:47:32 web:1447] 304 GET /register (192.168.32.17) 4.24ms
[I 120906 14:47:38 web:1447] 302 GET /twitter_login (192.168.32.17) 307.45ms
[I 120906 14:47:43 handlers:155] twitter oauth_token is YLBmTaG9o9i5JCtUbA6xnqvocVDxHkl78WjOUL6rRz4
[I 120906 14:47:44 handlers:163] twitter Auth worked
[I 120906 14:47:44 handlers:77] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:47:44 web:1447] 302 GET /twitter_login?oauth_token=YLBmTaG9o9i5JCtUbA6xnqvocVDxHkl78WjOUL6rRz4&oauth_verifier=r7e0g2GoGhzk7y03qHe0MzgG80jK3njC3WZIhepDyc (192.168.32.17) 569.64ms
[E 120906 14:47:44 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346910464|23ff0fe56b72c9c60613251ac7d9946975a9e2bd; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346910464|ed7ddb41043f82f5573e23c74edf3aa79db12c2c"', 'If-None-Match': '"8d6a22ba800c8dc9f9406c7fed99473c75a9c208"'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 213, in get
        user = get_current_user2()
    NameError: global name 'get_current_user2' is not defined
[E 120906 14:47:44 web:1447] 500 GET /hello (192.168.32.17) 1.58ms
[I 120906 14:48:19 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 14:48:19 web:2027] Invalid cookie signature 'InllcG4i|1346910464|23ff0fe56b72c9c60613251ac7d9946975a9e2bd'
[I 120906 14:48:19 web:1447] 302 GET /hello (192.168.32.17) 1.67ms
[W 120906 14:48:19 web:2027] Invalid cookie signature 'InllcG4i|1346910464|23ff0fe56b72c9c60613251ac7d9946975a9e2bd'
[I 120906 14:48:19 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.75ms
[W 120906 14:48:23 web:2027] Invalid cookie signature 'InllcG4i|1346910464|23ff0fe56b72c9c60613251ac7d9946975a9e2bd'
[I 120906 14:48:23 web:1447] 304 GET /register (192.168.32.17) 1.15ms
[I 120906 14:53:56 web:1447] 302 GET /twitter_login (192.168.32.17) 294.05ms
[I 120906 14:54:03 handlers:156] twitter oauth_token is ssF0bVNQSyr5Vr0ZzHTzvR1m1LlfWrHhDrKLW8YZq1w
[I 120906 14:54:04 handlers:164] twitter Auth worked
[I 120906 14:54:04 handlers:78] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:54:04 web:1447] 302 GET /twitter_login?oauth_token=ssF0bVNQSyr5Vr0ZzHTzvR1m1LlfWrHhDrKLW8YZq1w&oauth_verifier=lEUEhTqbPsIWYpSPRVZAd8gkEzGPP9dfThKqvZ4awE (192.168.32.17) 608.80ms
[E 120906 14:54:05 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346910844|fcb4ddde0401c39042d034484a13258b343cffd5"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 214, in get
        user = get_current_user2()
    NameError: global name 'get_current_user2' is not defined
[E 120906 14:54:05 web:1447] 500 GET /hello (192.168.32.17) 1.56ms
[I 120906 14:54:25 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 14:54:27 web:2027] Invalid cookie signature 'InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4'
[I 120906 14:54:27 web:1447] 302 GET /hello (192.168.32.17) 1.64ms
[W 120906 14:54:28 web:2027] Invalid cookie signature 'InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4'
[I 120906 14:54:28 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.75ms
[W 120906 14:54:31 web:2027] Invalid cookie signature 'InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4'
[I 120906 14:54:31 web:1447] 304 GET /register (192.168.32.17) 1.16ms
[W 120906 14:54:35 web:2027] Invalid cookie signature 'InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4'
[I 120906 14:54:35 web:1447] 200 GET /login (192.168.32.17) 1.01ms
[I 120906 14:54:39 web:1447] 302 GET /logout (192.168.32.17) 0.55ms
[W 120906 14:54:39 web:2027] Invalid cookie signature 'InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4'
[I 120906 14:54:39 web:1447] 304 GET /login (192.168.32.17) 1.05ms
[W 120906 14:54:42 web:2027] Invalid cookie signature 'InllcG4i|1346910844|93aec24dac10858d110b552f3484bc79f011ccb4'
[I 120906 14:54:42 web:1447] 304 GET /register (192.168.32.17) 1.02ms
[I 120906 14:54:50 web:1447] 302 GET /twitter_login (192.168.32.17) 295.27ms
[I 120906 14:54:55 handlers:156] twitter oauth_token is hHtQ1UVdidafgX3n4xCDfwzfePHooESHjRaA9DrnlzA
[I 120906 14:54:56 handlers:164] twitter Auth worked
[I 120906 14:54:56 handlers:78] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:54:56 web:1447] 302 GET /twitter_login?oauth_token=hHtQ1UVdidafgX3n4xCDfwzfePHooESHjRaA9DrnlzA&oauth_verifier=JznQLPrQMmae6Yrj5x0PgmMqOxF9l5b8b0qRbPtKk (192.168.32.17) 606.72ms
[E 120906 14:54:56 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346910896|a96b1ce9520bc3dc75a3ff9aceb34c9bb0ca3a64; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346910896|a6c2f113c4ce9705589ecba216c65496f29e6d88"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 214, in get
        user = self.get_current_user2()
    TypeError: 'dict' object is not callable
[E 120906 14:54:56 web:1447] 500 GET /hello (192.168.32.17) 1.72ms
[I 120906 14:55:37 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 14:56:34 web:2027] Invalid cookie signature 'InllcG4i|1346910896|a96b1ce9520bc3dc75a3ff9aceb34c9bb0ca3a64'
[I 120906 14:56:34 web:1447] 304 GET /register (192.168.32.17) 4.22ms
[I 120906 14:56:36 web:1447] 302 GET /twitter_login (192.168.32.17) 298.76ms
[I 120906 14:56:42 handlers:157] twitter oauth_token is eDaUJjYoag1JDloj7f2zypmQDYdMMC6mNps9GoPjc
[I 120906 14:56:43 handlers:165] twitter Auth worked
[I 120906 14:56:43 handlers:79] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:56:43 web:1447] 302 GET /twitter_login?oauth_token=eDaUJjYoag1JDloj7f2zypmQDYdMMC6mNps9GoPjc&oauth_verifier=TJZzecRKjqGncL7tks1oWrU0bmg1TKtctUjvyoejXo (192.168.32.17) 584.66ms
[I 120906 14:56:43 handlers:70] {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[E 120906 14:56:43 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346911003|0cefa181cd5ccecec9a267266a61de1436a7af5b; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346911003|0574aaacf03233fc95b5669e95ee9da832cd35e2"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 215, in get
        user = self.get_current_user2()
    TypeError: 'dict' object is not callable
[E 120906 14:56:43 web:1447] 500 GET /hello (192.168.32.17) 1.87ms
[I 120906 14:59:17 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 14:59:29 web:2027] Invalid cookie signature 'InllcG4i|1346911003|0cefa181cd5ccecec9a267266a61de1436a7af5b'
[I 120906 14:59:29 web:1447] 304 GET /register (192.168.32.17) 4.29ms
[I 120906 14:59:32 web:1447] 302 GET /twitter_login (192.168.32.17) 361.20ms
[I 120906 14:59:35 handlers:157] twitter oauth_token is Qm78rMeJ2oSLKDPhuTtydIm8gySuKw2gTd7Wu0DOM
[I 120906 14:59:36 handlers:165] twitter Auth worked
[I 120906 14:59:36 handlers:79] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 14:59:36 web:1447] 302 GET /twitter_login?oauth_token=Qm78rMeJ2oSLKDPhuTtydIm8gySuKw2gTd7Wu0DOM&oauth_verifier=tLPPpJT0jVW0pbEZ4KqO7PqoL2YRBgg8ByYDr3yZ5n8 (192.168.32.17) 590.56ms
[I 120906 14:59:36 handlers:70] this is in get_current_user2 is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'} 
[E 120906 14:59:36 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346911176|b43ed7acbe0d2199d1bfa7c414a935cb109cdb08; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346911176|61cd12a4df2267f9352a42a658dad64ee1757776"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 215, in get
        user = self.get_current_user2()
    TypeError: 'dict' object is not callable
[E 120906 14:59:36 web:1447] 500 GET /hello (192.168.32.17) 1.90ms
[I 120906 15:00:49 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:00:56 web:2027] Invalid cookie signature 'InllcG4i|1346911176|b43ed7acbe0d2199d1bfa7c414a935cb109cdb08'
[I 120906 15:00:56 web:1447] 304 GET /register (192.168.32.17) 4.23ms
[I 120906 15:00:58 web:1447] 302 GET /twitter_login (192.168.32.17) 285.61ms
[I 120906 15:01:04 handlers:157] twitter oauth_token is XNmGcdUWU9Bh0ehmWVaNB7Cr0efpXyXR6sCTsJzF3Y
[I 120906 15:01:04 handlers:165] twitter Auth worked
[I 120906 15:01:04 handlers:79] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:01:04 web:1447] 302 GET /twitter_login?oauth_token=XNmGcdUWU9Bh0ehmWVaNB7Cr0efpXyXR6sCTsJzF3Y&oauth_verifier=3lcPDZ2XVjQCfG8r6RgZFCxprmua1BNXsvdHF4z4iY4 (192.168.32.17) 587.66ms
[I 120906 15:01:04 handlers:70] this is in get_current_user2 is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'} 
[E 120906 15:01:04 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346911264|a7184de99bf21324391bd1a26026f6a6c17263cb; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346911264|1b1d7365b23b69233b2317bdd69c5ca003265025"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 216, in get
        user = self.get_current_user2()
    TypeError: 'dict' object is not callable
[E 120906 15:01:04 web:1447] 500 GET /hello (192.168.32.17) 1.90ms
[I 120906 15:05:04 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:05:12 web:2027] Invalid cookie signature 'InllcG4i|1346911264|a7184de99bf21324391bd1a26026f6a6c17263cb'
[I 120906 15:05:12 web:1447] 304 GET /register (192.168.32.17) 4.26ms
[I 120906 15:05:13 web:1447] 302 GET /twitter_login (192.168.32.17) 304.33ms
[I 120906 15:05:16 handlers:158] twitter oauth_token is Pyx7I6HjpiFbXrId5x5UddxzQizqyc0aRdxJyavuk
[I 120906 15:05:17 handlers:166] twitter Auth worked
[I 120906 15:05:17 handlers:80] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:05:17 web:1447] 302 GET /twitter_login?oauth_token=Pyx7I6HjpiFbXrId5x5UddxzQizqyc0aRdxJyavuk&oauth_verifier=TeqRRDTnLZg7pDZxhLkhch490mXzXlWtEF69H8wTgAw (192.168.32.17) 566.34ms
[I 120906 15:05:17 handlers:70] this is in get_current_user2 is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'} 
[I 120906 15:05:17 handlers:71] this is in get_current_user2 user type is <type 'dict'> 
[E 120906 15:05:17 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346911517|83b88cc4085896eed515ccf0b42f5c01be8b3aff; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346911517|77a08f45b1a7af84079cb123cdb599bfdfa258fc"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 217, in get
        user = self.get_current_user2()
    TypeError: 'dict' object is not callable
[E 120906 15:05:17 web:1447] 500 GET /hello (192.168.32.17) 2.02ms
[I 120906 15:07:15 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:07:18 web:2027] Invalid cookie signature 'InllcG4i|1346911517|83b88cc4085896eed515ccf0b42f5c01be8b3aff'
[I 120906 15:07:18 web:1447] 304 GET /register (192.168.32.17) 4.24ms
[I 120906 15:07:20 web:1447] 302 GET /twitter_login (192.168.32.17) 284.50ms
[I 120906 15:07:24 handlers:157] twitter oauth_token is UaCimV0xf5gabUKjH9YIESTpuD8NlaQvJgOFPNhI2w
[I 120906 15:07:24 handlers:165] twitter Auth worked
[I 120906 15:07:24 handlers:79] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:07:24 web:1447] 302 GET /twitter_login?oauth_token=UaCimV0xf5gabUKjH9YIESTpuD8NlaQvJgOFPNhI2w&oauth_verifier=MmbcWZSNUCwo9Xaeqezrg0l0jfovhX1FyFyDLFNtRI (192.168.32.17) 579.20ms
[I 120906 15:07:24 handlers:69] this is in get_current_user2 is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'} 
[I 120906 15:07:24 handlers:70] this is in get_current_user2 user type is <type 'dict'> 
[I 120906 15:07:24 handlers:220] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:07:24 web:1447] 200 GET /hello (192.168.32.17) 1.91ms
[I 120906 15:07:51 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:07:52 web:2027] Invalid cookie signature 'InllcG4i|1346911644|d21b95c9eb24a6520f590aaad97e7f38cb806557'
[I 120906 15:07:52 web:1447] 302 GET /hello (192.168.32.17) 1.65ms
[W 120906 15:07:52 web:2027] Invalid cookie signature 'InllcG4i|1346911644|d21b95c9eb24a6520f590aaad97e7f38cb806557'
[I 120906 15:07:52 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 4.19ms
[W 120906 15:07:56 web:2027] Invalid cookie signature 'InllcG4i|1346911644|d21b95c9eb24a6520f590aaad97e7f38cb806557'
[I 120906 15:07:56 web:1447] 304 GET /register (192.168.32.17) 1.17ms
[I 120906 15:08:02 web:1447] 302 GET /twitter_login (192.168.32.17) 279.85ms
[I 120906 15:08:05 handlers:157] twitter oauth_token is dL1ca7DvcXaiCnBzSgPixrHnfW55z8CwOjnk6CQW8o
[I 120906 15:08:05 handlers:165] twitter Auth worked
[I 120906 15:08:05 handlers:79] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:08:05 web:1447] 302 GET /twitter_login?oauth_token=dL1ca7DvcXaiCnBzSgPixrHnfW55z8CwOjnk6CQW8o&oauth_verifier=t5irqXfXXae0y5q11M34NER2QklqpiNQv6bNCNgugI (192.168.32.17) 677.31ms
[I 120906 15:08:05 handlers:69] this is in get_current_user2 is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'} 
[I 120906 15:08:05 handlers:70] this is in get_current_user2 user type is <type 'dict'> 
[I 120906 15:08:05 handlers:218] user1 is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:08:05 handlers:221] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:08:05 web:1447] 200 GET /hello (192.168.32.17) 2.03ms
[I 120906 15:09:01 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:09:07 web:2027] Invalid cookie signature 'InllcG4i|1346911685|a103bc8baad0d2e06b81773fbe6deab86433d1a5'
[I 120906 15:09:07 web:1447] 304 GET /register (192.168.32.17) 4.21ms
[I 120906 15:09:09 web:1447] 302 GET /twitter_login (192.168.32.17) 319.24ms
[I 120906 15:09:12 handlers:157] twitter oauth_token is K5B7aN0tKwIPn1UsKliOLmIYMBD0mqUn6qfksxMoQ
[I 120906 15:09:13 handlers:165] twitter Auth worked
[I 120906 15:09:13 handlers:79] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:09:13 web:1447] 302 GET /twitter_login?oauth_token=K5B7aN0tKwIPn1UsKliOLmIYMBD0mqUn6qfksxMoQ&oauth_verifier=HDdepLeWEqrHUIAxhcZY2mTgprxNmuPRkg0J3m8X4 (192.168.32.17) 594.19ms
[I 120906 15:09:13 handlers:220] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:09:13 handlers:221] username is yepn, image is https://si0.twimg.com/profile_images/65163362/my2_normal.jpg
[I 120906 15:09:13 web:1447] 304 GET /hello (192.168.32.17) 1.76ms
[I 120906 15:10:26 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:10:40 web:1447] 302 GET /logout (192.168.32.17) 1.07ms
[W 120906 15:10:40 web:2027] Invalid cookie signature 'InllcG4i|1346911753|f74f310740ba436f4cc4f8894de15770cff8340d'
[I 120906 15:10:40 web:1447] 304 GET /login (192.168.32.17) 3.80ms
[W 120906 15:10:45 web:2027] Invalid cookie signature 'InllcG4i|1346911753|f74f310740ba436f4cc4f8894de15770cff8340d'
[I 120906 15:10:45 web:1447] 304 GET /register (192.168.32.17) 1.15ms
[I 120906 15:10:47 web:1447] 302 GET /twitter_login (192.168.32.17) 306.90ms
[I 120906 15:10:50 handlers:159] twitter oauth_token is 9FJZKK2RT54o0Khh0IX168GCP6dAUQuykH1TqAbA
[I 120906 15:10:50 handlers:167] twitter Auth worked
[I 120906 15:10:50 handlers:81] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:10:50 web:1447] 302 GET /twitter_login?oauth_token=9FJZKK2RT54o0Khh0IX168GCP6dAUQuykH1TqAbA&oauth_verifier=TJSaKBpx21IImkv4NUAoLopD2iFY1VQaN3Q3lPdBx6Y (192.168.32.17) 577.04ms
[E 120906 15:10:50 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346911850|5a1ccb65d8f3ea4db786345627f606dbd7b670f2; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346911850|69002a1df9eb687bc30c89974204f0ce75a6fdf5"', 'If-None-Match': '"8d6a22ba800c8dc9f9406c7fed99473c75a9c208"'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 217, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 70, in get_current_user2
        _user["email"] = tornado.escape.json_decode(email)
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 15:10:50 web:1447] 500 GET /hello (192.168.32.17) 2.15ms
[I 120906 15:11:08 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:11:12 web:2027] Invalid cookie signature 'InllcG4i|1346911850|5a1ccb65d8f3ea4db786345627f606dbd7b670f2'
[I 120906 15:11:12 web:1447] 302 GET /hello (192.168.32.17) 1.65ms
[W 120906 15:11:12 web:2027] Invalid cookie signature 'InllcG4i|1346911850|5a1ccb65d8f3ea4db786345627f606dbd7b670f2'
[I 120906 15:11:12 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.80ms
[W 120906 15:11:16 web:2027] Invalid cookie signature 'InllcG4i|1346911850|5a1ccb65d8f3ea4db786345627f606dbd7b670f2'
[I 120906 15:11:16 web:1447] 304 GET /register (192.168.32.17) 1.14ms
[I 120906 15:11:18 web:1447] 302 GET /twitter_login (192.168.32.17) 294.85ms
[I 120906 15:11:20 handlers:159] twitter oauth_token is sU8U1dHuLvSiLhSUbwIcJqHsrabsjDEdlnL5n300jQ
[I 120906 15:11:21 handlers:167] twitter Auth worked
[I 120906 15:11:21 handlers:81] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:11:21 web:1447] 302 GET /twitter_login?oauth_token=sU8U1dHuLvSiLhSUbwIcJqHsrabsjDEdlnL5n300jQ&oauth_verifier=VJk8c4R6zlygNfg0BbqiukqNyVFr5iXkPGqFYMAe94 (192.168.32.17) 580.06ms
[I 120906 15:11:21 handlers:222] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:11:21 handlers:223] username is yepn, image is https://si0.twimg.com/profile_images/65163362/my2_normal.jpg
[I 120906 15:11:21 web:1447] 200 GET /hello (192.168.32.17) 1.74ms
[I 120906 15:16:01 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:16:04 web:1447] 302 GET /logout (192.168.32.17) 1.08ms
[W 120906 15:16:04 web:2027] Invalid cookie signature 'InllcG4i|1346911881|df4919afb25aa1548fbf402ecf8e47a87bb1bdf7'
[I 120906 15:16:04 web:1447] 304 GET /login (192.168.32.17) 3.80ms
[W 120906 15:16:12 web:2027] Invalid cookie signature 'InllcG4i|1346911881|df4919afb25aa1548fbf402ecf8e47a87bb1bdf7'
[I 120906 15:16:12 web:1447] 304 GET /register (192.168.32.17) 1.14ms
[I 120906 15:16:16 web:1447] 302 GET /twitter_login (192.168.32.17) 292.22ms
[I 120906 15:16:20 handlers:163] twitter oauth_token is fUImvvRo5fJgdoyyoEB6oGzrMEvFOcKUvsMYNXdeM
[I 120906 15:16:20 handlers:171] twitter Auth worked
[I 120906 15:16:20 handlers:85] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:16:20 web:1447] 302 GET /twitter_login?oauth_token=fUImvvRo5fJgdoyyoEB6oGzrMEvFOcKUvsMYNXdeM&oauth_verifier=DfOjz1PwiXr7QlDYMT6GigXvLuBZRA6t3MNKcoEM (192.168.32.17) 583.15ms
[I 120906 15:16:20 handlers:226] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:16:20 handlers:227] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:16:20 web:1447] 304 GET /hello (192.168.32.17) 1.70ms
[I 120906 15:17:15 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:17:19 web:1447] 302 GET /logout (192.168.32.17) 1.07ms
[W 120906 15:17:19 web:2027] Invalid cookie signature 'InllcG4i|1346912180|9a63406ec9bfa9d8692cd528b74ecec972b63d8f'
[I 120906 15:17:19 web:1447] 304 GET /login (192.168.32.17) 3.78ms
[W 120906 15:17:21 web:2027] Invalid cookie signature 'InllcG4i|1346912180|9a63406ec9bfa9d8692cd528b74ecec972b63d8f'
[I 120906 15:17:21 web:1447] 304 GET /register (192.168.32.17) 1.17ms
[I 120906 15:17:23 web:1447] 302 GET /twitter_login (192.168.32.17) 308.78ms
[I 120906 15:17:26 handlers:163] twitter oauth_token is 7F9yVKJd3KPBXhwyaNsBR9LdfbLnm0VkMkCaLshpo
[I 120906 15:17:26 handlers:171] twitter Auth worked
[I 120906 15:17:26 handlers:85] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:17:26 web:1447] 302 GET /twitter_login?oauth_token=7F9yVKJd3KPBXhwyaNsBR9LdfbLnm0VkMkCaLshpo&oauth_verifier=7zjJIxz9g3cZ6mchVeUPnCkjdLUDKZ92xBuxslgZojs (192.168.32.17) 580.16ms
[E 120906 15:17:26 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346912246|786b12342b0ed119b3014d9449f7d09bf03616c3; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346912246|695131b76125575c15f194b66c587609293b756a"', 'If-None-Match': '"8d6a22ba800c8dc9f9406c7fed99473c75a9c208"'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 221, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 77, in get_current_user2
        return tornado.escape.json_decode(_user)
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 210, in to_basestring
        assert isinstance(value, bytes)
    AssertionError
[E 120906 15:17:26 web:1447] 500 GET /hello (192.168.32.17) 1.89ms
[I 120906 15:18:00 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:18:01 web:2027] Invalid cookie signature 'InllcG4i|1346912246|786b12342b0ed119b3014d9449f7d09bf03616c3'
[I 120906 15:18:01 web:1447] 302 GET /hello (192.168.32.17) 1.66ms
[W 120906 15:18:01 web:2027] Invalid cookie signature 'InllcG4i|1346912246|786b12342b0ed119b3014d9449f7d09bf03616c3'
[I 120906 15:18:01 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.76ms
[W 120906 15:18:04 web:2027] Invalid cookie signature 'InllcG4i|1346912246|786b12342b0ed119b3014d9449f7d09bf03616c3'
[I 120906 15:18:04 web:1447] 304 GET /register (192.168.32.17) 1.15ms
[I 120906 15:18:06 web:1447] 302 GET /twitter_login (192.168.32.17) 289.11ms
[I 120906 15:18:08 handlers:163] twitter oauth_token is hA2pLqGxgPA4OH2JnGfzYbNGEUO05OcH4zU950i4
[I 120906 15:18:09 handlers:171] twitter Auth worked
[I 120906 15:18:09 handlers:85] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:18:09 web:1447] 302 GET /twitter_login?oauth_token=hA2pLqGxgPA4OH2JnGfzYbNGEUO05OcH4zU950i4&oauth_verifier=7jPgQNmgxzUPs2NhrYamDI6PVaJh4TmjMVwmya8KKE (192.168.32.17) 587.53ms
[I 120906 15:18:09 handlers:227] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:18:09 web:1447] 200 GET /hello (192.168.32.17) 1.56ms
[I 120906 15:18:52 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:18:55 web:1447] 302 GET /logout (192.168.32.17) 1.07ms
[W 120906 15:18:55 web:2027] Invalid cookie signature 'InllcG4i|1346912289|c1050a526469de6d6f48093109e10189adc10772'
[I 120906 15:18:55 web:1447] 304 GET /login (192.168.32.17) 3.75ms
[W 120906 15:18:56 web:2027] Invalid cookie signature 'InllcG4i|1346912289|c1050a526469de6d6f48093109e10189adc10772'
[I 120906 15:18:56 web:1447] 304 GET /register (192.168.32.17) 1.13ms
[I 120906 15:19:02 web:1447] 302 GET /twitter_login (192.168.32.17) 302.50ms
[I 120906 15:19:05 handlers:163] twitter oauth_token is wbeqiVlXDvYSnAUUrmljV3IJb5RROt3vYtJC8zYx8
[I 120906 15:19:06 handlers:171] twitter Auth worked
[I 120906 15:19:06 handlers:85] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:19:06 web:1447] 302 GET /twitter_login?oauth_token=wbeqiVlXDvYSnAUUrmljV3IJb5RROt3vYtJC8zYx8&oauth_verifier=lQkGUIXfkS5tF4GX6pRnUAKdEioDgLWW2G4jxLYBd0 (192.168.32.17) 596.58ms
[I 120906 15:19:06 handlers:227] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:19:06 web:1447] 200 GET /hello (192.168.32.17) 1.52ms
[W 120906 15:19:06 web:1447] 404 GET /%22https://si0.twimg.com/profile_images/65163362/my2_normal.jpg%22 (192.168.32.17) 0.83ms
[I 120906 15:19:44 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:19:46 web:2027] Invalid cookie signature 'InllcG4i|1346912346|d8ee84d3d1dc4285e56908d56b2dcec076f7400f'
[I 120906 15:19:46 web:1447] 302 GET /hello (192.168.32.17) 1.66ms
[W 120906 15:19:46 web:2027] Invalid cookie signature 'InllcG4i|1346912346|d8ee84d3d1dc4285e56908d56b2dcec076f7400f'
[I 120906 15:19:46 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.74ms
[W 120906 15:19:48 web:2027] Invalid cookie signature 'InllcG4i|1346912346|d8ee84d3d1dc4285e56908d56b2dcec076f7400f'
[I 120906 15:19:48 web:1447] 304 GET /register (192.168.32.17) 1.14ms
[I 120906 15:19:52 web:1447] 302 GET /twitter_login (192.168.32.17) 315.37ms
[I 120906 15:19:57 handlers:163] twitter oauth_token is qin8G32qUcTObeAbWiFoAY7jGaMWlrf49PV74pw3D8
[I 120906 15:19:57 handlers:171] twitter Auth worked
[I 120906 15:19:57 handlers:85] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:19:57 web:1447] 302 GET /twitter_login?oauth_token=qin8G32qUcTObeAbWiFoAY7jGaMWlrf49PV74pw3D8&oauth_verifier=s0vKjoc7mLtzi4CQUlP5HptATr7UXi1KzacLk6Dw (192.168.32.17) 577.67ms
[E 120906 15:19:57 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346912397|2b5669e364582a9ae95ecb5f6ef8b7ee34ad22e4; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346912397|d68b0d269a535ac8618dede5d89592e932c1333f"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 221, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 77, in get_current_user2
        return tornado.escape.json_decode(_user)
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 210, in to_basestring
        assert isinstance(value, bytes)
    AssertionError
[E 120906 15:19:57 web:1447] 500 GET /hello (192.168.32.17) 1.88ms
[I 120906 15:40:19 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:40:26 web:2027] Invalid cookie signature 'InllcG4i|1346912397|2b5669e364582a9ae95ecb5f6ef8b7ee34ad22e4'
[I 120906 15:40:26 web:1447] 304 GET /register (192.168.32.17) 4.31ms
[I 120906 15:40:28 web:1447] 302 GET /twitter_login (192.168.32.17) 308.13ms
[I 120906 15:40:31 handlers:164] twitter oauth_token is Mxmjc7XKGcVA3xagRTHHsp6Lb0nh3pbYBDIuF9N8Dw8
[I 120906 15:40:32 handlers:172] twitter Auth worked
[I 120906 15:40:32 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:40:32 web:1447] 302 GET /twitter_login?oauth_token=Mxmjc7XKGcVA3xagRTHHsp6Lb0nh3pbYBDIuF9N8Dw8&oauth_verifier=QQnq9us4jCj059IVMUdNIcAjUwn9rd75hupICx8 (192.168.32.17) 605.18ms
[I 120906 15:40:32 handlers:69] "yepn"
[E 120906 15:40:32 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346913632|dcf19c839a9f72a7775fdfdcdbb0669af777a3fa; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346913632|9315e7c78aa5489b5264c866573fe87912b14439"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 78, in get_current_user2
        return tornado.escape.json_decode(_user)
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 210, in to_basestring
        assert isinstance(value, bytes)
    AssertionError
[E 120906 15:40:32 web:1447] 500 GET /hello (192.168.32.17) 2.05ms
[I 120906 15:40:46 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:40:53 web:2027] Invalid cookie signature 'InllcG4i|1346913632|dcf19c839a9f72a7775fdfdcdbb0669af777a3fa'
[I 120906 15:40:53 web:1447] 304 GET /register (192.168.32.17) 4.21ms
[I 120906 15:40:59 web:1447] 302 GET /twitter_login (192.168.32.17) 308.32ms
[I 120906 15:41:03 handlers:164] twitter oauth_token is v6J7Xu99GVgnzfjksA9bbLIooOhGadTmLgS8ovekk
[I 120906 15:41:04 handlers:172] twitter Auth worked
[I 120906 15:41:04 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:41:04 web:1447] 302 GET /twitter_login?oauth_token=v6J7Xu99GVgnzfjksA9bbLIooOhGadTmLgS8ovekk&oauth_verifier=hsQccomr3kAwBbupzhEVJIrgWP5PpYmGYglpSbJDI (192.168.32.17) 576.60ms
[E 120906 15:41:04 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346913664|a484a20fdfa39997c58ffb20f6370cd9cd06a61f; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346913664|4e2d6402914143b20cd015ce5dfb14fdb0831d07"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 69, in get_current_user2
        logging.info(typ(_user["username"]))
    NameError: global name 'typ' is not defined
[E 120906 15:41:04 web:1447] 500 GET /hello (192.168.32.17) 1.70ms
[I 120906 15:41:13 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:41:18 web:2027] Invalid cookie signature 'InllcG4i|1346913664|a484a20fdfa39997c58ffb20f6370cd9cd06a61f'
[I 120906 15:41:18 web:1447] 304 GET /register (192.168.32.17) 4.19ms
[I 120906 15:41:20 web:1447] 302 GET /twitter_login (192.168.32.17) 281.36ms
[I 120906 15:41:22 handlers:164] twitter oauth_token is Y3rfbTEPqQIMVerLhtC71eJzuvUYWdYcwQUeZrf0mBU
[I 120906 15:41:23 handlers:172] twitter Auth worked
[I 120906 15:41:23 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:41:23 web:1447] 302 GET /twitter_login?oauth_token=Y3rfbTEPqQIMVerLhtC71eJzuvUYWdYcwQUeZrf0mBU&oauth_verifier=c4xaurkywjIj8vsLrTbwcptk8yLQzkIOsXjOgbksXqo (192.168.32.17) 584.78ms
[I 120906 15:41:23 handlers:69] <type 'str'>
[E 120906 15:41:23 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346913683|dced7119d5c945bbc11e3f9eb79f2706a97e3759; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346913683|112b7fd0b67634df8d362c9923850bf8dd7b4d94"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 78, in get_current_user2
        return tornado.escape.json_decode(_user)
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 210, in to_basestring
        assert isinstance(value, bytes)
    AssertionError
[E 120906 15:41:23 web:1447] 500 GET /hello (192.168.32.17) 2.03ms
[I 120906 15:41:49 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:42:05 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:42:11 web:2027] Invalid cookie signature 'InllcG4i|1346913683|dced7119d5c945bbc11e3f9eb79f2706a97e3759'
[I 120906 15:42:11 web:1447] 304 GET /register (192.168.32.17) 4.23ms
[I 120906 15:42:13 web:1447] 302 GET /twitter_login (192.168.32.17) 309.80ms
[I 120906 15:42:18 handlers:164] twitter oauth_token is auLdOccWfFQDd9lV0IyQLVJNgEHrCVNGAZen4iqiI3c
[I 120906 15:42:19 handlers:172] twitter Auth worked
[I 120906 15:42:19 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:42:19 web:1447] 302 GET /twitter_login?oauth_token=auLdOccWfFQDd9lV0IyQLVJNgEHrCVNGAZen4iqiI3c&oauth_verifier=YMpwCZHfJvBE9Pb2Kr8TfPo2NxRULD6a2lJDT1P5jE (192.168.32.17) 574.63ms
[I 120906 15:42:19 handlers:69] "yepn"
[E 120906 15:42:19 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346913739|64d9ec67d5e77b8ce37a1eef3580153961a7e241; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346913739|777283ff6c2023bc9cbc48463961604700077733"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 78, in get_current_user2
        return tornado.escape.json_decode(_user)
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 210, in to_basestring
        assert isinstance(value, bytes)
    AssertionError
[E 120906 15:42:19 web:1447] 500 GET /hello (192.168.32.17) 2.03ms
[I 120906 15:43:02 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:43:07 web:2027] Invalid cookie signature 'InllcG4i|1346913739|64d9ec67d5e77b8ce37a1eef3580153961a7e241'
[I 120906 15:43:07 web:1447] 304 GET /register (192.168.32.17) 4.22ms
[I 120906 15:43:11 web:1447] 302 GET /twitter_login (192.168.32.17) 308.10ms
[I 120906 15:43:15 handlers:163] twitter oauth_token is 0ClC2o5yovlOy8IlPhtP72JQSdeIMmyAZY7m0UnRXY
[I 120906 15:43:15 handlers:171] twitter Auth worked
[I 120906 15:43:15 handlers:85] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:43:15 web:1447] 302 GET /twitter_login?oauth_token=0ClC2o5yovlOy8IlPhtP72JQSdeIMmyAZY7m0UnRXY&oauth_verifier=8sSE8T67eEuFyX3Wvr5UEp6NmGYCBAPwNrmzyOoj0YQ (192.168.32.17) 594.64ms
[I 120906 15:43:15 handlers:227] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:43:15 web:1447] 200 GET /hello (192.168.32.17) 1.55ms
[I 120906 15:44:05 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:44:13 web:1447] 302 GET /logout (192.168.32.17) 1.07ms
[W 120906 15:44:13 web:2027] Invalid cookie signature 'InllcG4i|1346913795|0d632a84a4f3294715993bee8e2dcda7c67c975a'
[I 120906 15:44:13 web:1447] 304 GET /login (192.168.32.17) 3.78ms
[W 120906 15:44:14 web:2027] Invalid cookie signature 'InllcG4i|1346913795|0d632a84a4f3294715993bee8e2dcda7c67c975a'
[I 120906 15:44:14 web:1447] 304 GET /register (192.168.32.17) 1.14ms
[I 120906 15:44:18 web:1447] 302 GET /twitter_login (192.168.32.17) 312.04ms
[I 120906 15:44:21 handlers:164] twitter oauth_token is Msq2VCOzrBIvqW82GCN6WUfFTf1rD4Bw3TxQfV7Dgg
[I 120906 15:44:22 handlers:172] twitter Auth worked
[I 120906 15:44:22 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:44:22 web:1447] 302 GET /twitter_login?oauth_token=Msq2VCOzrBIvqW82GCN6WUfFTf1rD4Bw3TxQfV7Dgg&oauth_verifier=5G4nGYWOSc136kGUqtScc0fiibq76bOKfFv2rUkTGes (192.168.32.17) 566.07ms
[E 120906 15:44:22 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346913862|7663402818013bab96b67b674248fc77f872e4e2; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346913862|4660a99d08a450c5f11c586e7ad9e883276275c8"', 'If-None-Match': '"8d6a22ba800c8dc9f9406c7fed99473c75a9c208"'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 68, in get_current_user2
        logging.infod(_user)
    AttributeError: 'module' object has no attribute 'infod'
[E 120906 15:44:22 web:1447] 500 GET /hello (192.168.32.17) 1.72ms
[I 120906 15:44:31 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:44:33 web:2027] Invalid cookie signature 'InllcG4i|1346913862|7663402818013bab96b67b674248fc77f872e4e2'
[I 120906 15:44:33 web:1447] 302 GET /hello (192.168.32.17) 1.65ms
[W 120906 15:44:33 web:2027] Invalid cookie signature 'InllcG4i|1346913862|7663402818013bab96b67b674248fc77f872e4e2'
[I 120906 15:44:33 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.74ms
[W 120906 15:44:36 web:2027] Invalid cookie signature 'InllcG4i|1346913862|7663402818013bab96b67b674248fc77f872e4e2'
[I 120906 15:44:36 web:1447] 304 GET /register (192.168.32.17) 1.18ms
[I 120906 15:44:42 web:1447] 302 GET /twitter_login (192.168.32.17) 300.55ms
[I 120906 15:44:44 handlers:164] twitter oauth_token is 1JEjbfVai7U1koZd4x7lGnUHuvHskvIbtYqKKIg6cwE
[I 120906 15:44:44 handlers:172] twitter Auth worked
[I 120906 15:44:44 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:44:44 web:1447] 302 GET /twitter_login?oauth_token=1JEjbfVai7U1koZd4x7lGnUHuvHskvIbtYqKKIg6cwE&oauth_verifier=rp6CWUnZykNz9bdT5Zkgi8tMiArGb6Z6c8vK6brMJiQ (192.168.32.17) 592.54ms
[I 120906 15:44:44 handlers:68] {'username': '"yepn"', 'image': '"https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"', 'email': None}
[I 120906 15:44:44 handlers:228] username is "yepn", image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:44:44 web:1447] 200 GET /hello (192.168.32.17) 1.74ms
[I 120906 15:46:18 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[I 120906 15:46:25 web:1447] 302 GET /logout (192.168.32.17) 1.06ms
[W 120906 15:46:25 web:2027] Invalid cookie signature 'InllcG4i|1346913884|3948158c7c79953df71251b4e7bc82dc563daa07'
[I 120906 15:46:25 web:1447] 304 GET /login (192.168.32.17) 3.79ms
[W 120906 15:46:29 web:2027] Invalid cookie signature 'InllcG4i|1346913884|3948158c7c79953df71251b4e7bc82dc563daa07'
[I 120906 15:46:29 web:1447] 304 GET /register (192.168.32.17) 1.14ms
[I 120906 15:46:33 web:1447] 302 GET /twitter_login (192.168.32.17) 343.99ms
[I 120906 15:46:38 handlers:164] twitter oauth_token is gvpNlh9GdWmeeBMwhvWOzuo28OUyK5yxQQOob9YPfc
[I 120906 15:46:39 handlers:172] twitter Auth worked
[I 120906 15:46:39 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:46:39 web:1447] 302 GET /twitter_login?oauth_token=gvpNlh9GdWmeeBMwhvWOzuo28OUyK5yxQQOob9YPfc&oauth_verifier=BzmKDs7CZlFnEv4RVEypSdHPtaGT7hzBy3NPpJTtS0 (192.168.32.17) 580.35ms
[E 120906 15:46:39 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346913999|66081c414e139da521a3e6d57c18454b21189938; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346913999|a1b0ef5676b39cf60cb336e818f257fdfa219bcd"', 'If-None-Match': '"8d6a22ba800c8dc9f9406c7fed99473c75a9c208"'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 68, in get_current_user2
        logging.info(tornado.escape.json_decode(_user))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 210, in to_basestring
        assert isinstance(value, bytes)
    AssertionError
[E 120906 15:46:39 web:1447] 500 GET /hello (192.168.32.17) 1.86ms
[I 120906 15:49:03 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:49:09 web:2027] Invalid cookie signature 'InllcG4i|1346913999|66081c414e139da521a3e6d57c18454b21189938'
[I 120906 15:49:09 web:1447] 304 GET /register (192.168.32.17) 4.22ms
[I 120906 15:49:11 web:1447] 302 GET /twitter_login (192.168.32.17) 301.21ms
[I 120906 15:49:15 handlers:164] twitter oauth_token is NwZi0Lo0Wiqf0w3iz4hB9DO8wQtSH1E8oHOM0fBAssc
[I 120906 15:49:15 handlers:172] twitter Auth worked
[I 120906 15:49:15 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:49:15 web:1447] 302 GET /twitter_login?oauth_token=NwZi0Lo0Wiqf0w3iz4hB9DO8wQtSH1E8oHOM0fBAssc&oauth_verifier=yKd1pEn8aVs9sYPl5OfiEilNLx7Fj8ibEojELTMWlZs (192.168.32.17) 578.30ms
[I 120906 15:49:16 handlers:68] {'username': u'yepn', 'image': '"https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"', 'email': None}
[I 120906 15:49:16 handlers:228] username is yepn, image is "https://si0.twimg.com/profile_images/65163362/my2_normal.jpg"
[I 120906 15:49:16 web:1447] 200 GET /hello (192.168.32.17) 1.78ms
[I 120906 15:51:47 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:51:57 web:2027] Invalid cookie signature 'InllcG4i|1346914155|de998f4cb3e414be8f06218eefd51d767cd8d5f5'
[I 120906 15:51:57 web:1447] 304 GET /register (192.168.32.17) 4.19ms
[I 120906 15:51:59 web:1447] 302 GET /twitter_login (192.168.32.17) 296.47ms
[I 120906 15:52:02 handlers:164] twitter oauth_token is ZlU0gDXExDunDVOYozH8kiIPasRrwfsFitx0iilGrh8
[I 120906 15:52:03 handlers:172] twitter Auth worked
[I 120906 15:52:03 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:52:03 web:1447] 302 GET /twitter_login?oauth_token=ZlU0gDXExDunDVOYozH8kiIPasRrwfsFitx0iilGrh8&oauth_verifier=mO0uGEzY2fgizSqJ3bOQ8vLuca734GOhz18assEVEY (192.168.32.17) 573.85ms
[E 120906 15:52:03 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346914323|ab815ab80ad8d313298b850fe5d2fd4a43a59e76; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346914323|1cb38a74c0472401a710c25cc99e2f3234d8641d"', 'If-None-Match': '"8d6a22ba800c8dc9f9406c7fed99473c75a9c208"'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 67, in get_current_user2
        _user["email"] = tornado.escape.json_decode(self.get_secure_cookie("email",None))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 15:52:03 web:1447] 500 GET /hello (192.168.32.17) 2.17ms
[I 120906 15:52:23 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:52:26 web:2027] Invalid cookie signature 'InllcG4i|1346914323|ab815ab80ad8d313298b850fe5d2fd4a43a59e76'
[I 120906 15:52:26 web:1447] 302 GET /hello (192.168.32.17) 1.65ms
[W 120906 15:52:26 web:2027] Invalid cookie signature 'InllcG4i|1346914323|ab815ab80ad8d313298b850fe5d2fd4a43a59e76'
[I 120906 15:52:26 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.76ms
[W 120906 15:52:29 web:2027] Invalid cookie signature 'InllcG4i|1346914323|ab815ab80ad8d313298b850fe5d2fd4a43a59e76'
[I 120906 15:52:29 web:1447] 304 GET /register (192.168.32.17) 1.15ms
[I 120906 15:52:32 web:1447] 302 GET /twitter_login (192.168.32.17) 294.00ms
[I 120906 15:52:34 handlers:164] twitter oauth_token is AwczC5Et86WuR8nIsOiHppJEAklVM6NtLmNfD1zO8
[I 120906 15:52:35 handlers:172] twitter Auth worked
[I 120906 15:52:35 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:52:35 web:1447] 302 GET /twitter_login?oauth_token=AwczC5Et86WuR8nIsOiHppJEAklVM6NtLmNfD1zO8&oauth_verifier=LuPnqYAh2j9DlHK4vfqxHCV2UMdW9cHtZ2wx5kjxhQ (192.168.32.17) 592.72ms
[E 120906 15:52:35 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346914355|c05950b70b332a953b6e59f31f3f135bf3ebe31a; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346914355|e0e312e5fa55c0b464c7df8bbc95e655aa1a4626"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 65, in get_current_user2
        _user["username"] = tornado.escape.json_decode(self.get_secure_cookie("username",value=""))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 15:52:35 web:1447] 500 GET /hello (192.168.32.17) 1.94ms
[I 120906 15:52:49 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 15:53:05 web:2027] Invalid cookie signature 'InllcG4i|1346914355|c05950b70b332a953b6e59f31f3f135bf3ebe31a'
[I 120906 15:53:05 web:1447] 304 GET /register (192.168.32.17) 3.96ms
[I 120906 15:53:07 web:1447] 302 GET /twitter_login (192.168.32.17) 304.60ms
[I 120906 15:53:11 handlers:164] twitter oauth_token is KMZ21Kw1IJiPFEZYTyQqH4UvNaFO0CicRL2fen59i0
[I 120906 15:53:11 handlers:172] twitter Auth worked
[I 120906 15:53:11 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 15:53:11 web:1447] 302 GET /twitter_login?oauth_token=KMZ21Kw1IJiPFEZYTyQqH4UvNaFO0CicRL2fen59i0&oauth_verifier=bHbyZ87gsFEFMBE0gRy8ehcyJJceDvSTRJHIVAKNNk (192.168.32.17) 562.80ms
[E 120906 15:53:11 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346914391|66ab0cab8f15e4e1a03a8559614ec0f1ae93af6e; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346914391|f10f7213dd6e0202d5a3d383ea190e19e47b8b80"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 67, in get_current_user2
        _user["email"] = tornado.escape.json_decode(self.get_secure_cookie("email"))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 15:53:11 web:1447] 500 GET /hello (192.168.32.17) 2.15ms
[E 120906 16:16:41 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346914391|66ab0cab8f15e4e1a03a8559614ec0f1ae93af6e; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346914391|f10f7213dd6e0202d5a3d383ea190e19e47b8b80"', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 67, in get_current_user2
        _user["email"] = tornado.escape.json_decode(self.get_secure_cookie("email"))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 16:16:41 web:1447] 500 GET /hello (192.168.32.17) 1.65ms
[E 120906 16:16:42 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0', 'Connection': 'keep-alive', 'Cookie': 'username=InllcG4i|1346914391|66ab0cab8f15e4e1a03a8559614ec0f1ae93af6e; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346914391|f10f7213dd6e0202d5a3d383ea190e19e47b8b80"', 'Cache-Control': 'max-age=0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 67, in get_current_user2
        _user["email"] = tornado.escape.json_decode(self.get_secure_cookie("email"))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 16:16:42 web:1447] 500 GET /hello (192.168.32.17) 1.12ms
[I 120906 16:16:44 web:1447] 304 GET /register (192.168.32.17) 1.03ms
[I 120906 16:25:17 web:1447] 304 GET /register (192.168.32.17) 0.92ms
[I 120906 16:25:31 web:1447] 302 GET /twitter_login (192.168.32.17) 290.93ms
[I 120906 16:25:35 handlers:164] twitter oauth_token is 2nszEa27ClgETtBzlqrGDWaDyFVIFUpZwX0KjvcSp7g
[I 120906 16:25:35 handlers:172] twitter Auth worked
[I 120906 16:25:35 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 16:25:35 web:1447] 302 GET /twitter_login?oauth_token=2nszEa27ClgETtBzlqrGDWaDyFVIFUpZwX0KjvcSp7g&oauth_verifier=vTLIvRo7iNq2jowq5IlfHD4ZuAqkCUGP8adoQOBLY (192.168.32.17) 576.06ms
[E 120906 16:25:35 web:1064] Uncaught exception GET /hello (192.168.32.17)
    HTTPRequest(protocol='http', host='10.24.94.36:8888', method='GET', uri='/hello', version='HTTP/1.1', remote_ip='192.168.32.17', body='', headers={'Connection': 'keep-alive', 'Accept-Language': 'ja,en-us;q=0.7,en;q=0.3', 'Accept-Encoding': 'gzip, deflate', 'Cookie': 'username=InllcG4i|1346916335|36a6e07972ff1da3c2951f41d04d4dd27e05c40a; image="Imh0dHBzOi8vc2kwLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy82NTE2MzM2Mi9teTJfbm9ybWFsLmpwZyI=|1346916335|dd9001d54838b1649dbd5cac2f58847680f98960"', 'Host': '10.24.94.36:8888', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'User-Agent': 'Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20100101 Firefox/15.0'})
    Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1021, in _execute
        getattr(self, self.request.method.lower())(*args, **kwargs)
      File "/usr/lib/python2.6/site-packages/tornado/web.py", line 1794, in wrapper
        return method(self, *args, **kwargs)
      File "/root/bipapa/handlers/handlers.py", line 222, in get
        user = self.get_current_user2()
      File "/root/bipapa/handlers/handlers.py", line 67, in get_current_user2
        _user["email"] = tornado.escape.json_decode(self.get_secure_cookie("email"))
      File "/usr/lib/python2.6/site-packages/tornado/escape.py", line 95, in json_decode
        return _json_decode(to_basestring(value))
      File "/usr/lib64/python2.6/json/__init__.py", line 307, in loads
        return _default_decoder.decode(s)
      File "/usr/lib64/python2.6/json/decoder.py", line 319, in decode
        obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    TypeError: expected string or buffer
[E 120906 16:25:35 web:1447] 500 GET /hello (192.168.32.17) 2.05ms
[I 120906 16:25:53 autoreload:175] /root/bipapa/handlers/handlers.py modified; restarting server
[W 120906 16:25:54 web:2027] Invalid cookie signature 'InllcG4i|1346916335|36a6e07972ff1da3c2951f41d04d4dd27e05c40a'
[I 120906 16:25:54 web:1447] 302 GET /hello (192.168.32.17) 1.66ms
[W 120906 16:25:54 web:2027] Invalid cookie signature 'InllcG4i|1346916335|36a6e07972ff1da3c2951f41d04d4dd27e05c40a'
[I 120906 16:25:54 web:1447] 304 GET /login?next=%2Fhello (192.168.32.17) 3.80ms
[W 120906 16:25:57 web:2027] Invalid cookie signature 'InllcG4i|1346916335|36a6e07972ff1da3c2951f41d04d4dd27e05c40a'
[I 120906 16:25:57 web:1447] 304 GET /register (192.168.32.17) 1.17ms
[I 120906 16:26:01 web:1447] 302 GET /twitter_login (192.168.32.17) 293.68ms
[I 120906 16:26:03 handlers:164] twitter oauth_token is Cy3phetO3nOhp19W4lHTjyQ4RNmwe76dCbOY0UaQ00c
[I 120906 16:26:03 handlers:172] twitter Auth worked
[I 120906 16:26:03 handlers:86] this is in set_current_user is {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 16:26:03 web:1447] 302 GET /twitter_login?oauth_token=Cy3phetO3nOhp19W4lHTjyQ4RNmwe76dCbOY0UaQ00c&oauth_verifier=V5RxzSzaeZGZrAbF1zyTBQuEKnS1zInf0idOIkV64ck (192.168.32.17) 587.83ms
[I 120906 16:26:03 handlers:68] {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 16:26:03 handlers:228] username is yepn, image is https://si0.twimg.com/profile_images/65163362/my2_normal.jpg
[I 120906 16:26:03 web:1447] 200 GET /hello (192.168.32.17) 1.79ms
[I 120906 16:26:11 handlers:68] {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120906 16:26:11 handlers:228] username is yepn, image is https://si0.twimg.com/profile_images/65163362/my2_normal.jpg
[I 120906 16:26:11 web:1447] 304 GET /hello (192.168.32.17) 1.74ms
[I 120907 17:58:07 handlers:68] {'username': u'yepn', 'image': u'https://si0.twimg.com/profile_images/65163362/my2_normal.jpg'}
[I 120907 17:58:07 handlers:228] username is yepn, image is https://si0.twimg.com/profile_images/65163362/my2_normal.jpg
[I 120907 17:58:07 web:1447] 304 GET /hello (192.168.32.17) 2.19ms
